board/piksiv3/rootfs-overlay/etc/init.d/S45zmq_adapter_gps0

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+name="zmq_adapter_gps0"
+cmd="zmq_adapter --stdio -s '>tcp://127.0.0.1:44030' | socat pty,raw,link=/dev/gps0 fd:0"
```

Comment:
I think later we can consolidate this into the new nmea_daemon.

Reply:
I agree